guidelines/templates/README.md

```diff
@@ -15,6 +15,7 @@ The table provides an overview of the template names, the repository they refer
 | [`community`](https://github.com/kyma-project/community) | [`ADR.md`](./resources/ADR.md) | Use the template for ADRs.
 | All | [`README.md`](../../guidelines/internal-guidelines/repository-template/template/README.md) | Use the template for the main README.md document in any repository within the Kyma organization. |
 | [`kyma/docs`](https://github.com/kyma-project/kyma/tree/master/docs) | [`cli-reference.md`](./resources/cli-reference.md) | Use the template to describe the syntax and the use of CLI commands for a given component. |
+| [`kyma/docs`](https://github.com/kyma-project/kyma/tree/master/docs) | [`custom-resource.md`](./resources/custom-resource.md) | Use the template to describe a custom resource (CR) and its elements. |
```

Comment:
Capitalize "custom resource"

Reply:
It was my advice to not capitalize as it is not an invented name (however `nazwa wlasna` is called in english in this context). Is is just a resource, but custom one. Custom Resource Definition is an invented name
https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/

Reply:
Sure, makes sense, as long as you start capitalizing `Resource` ;)

Reply:
Ok, I'd stay with "custom resource" without capitalization. As all changes are applied and/or addressed, I'm finally merging this PR 😉